cosima_cookbook/explore.py

```diff
@@ -1,15 +1,18 @@
 import re
+import warnings
 
 from ipywidgets import HTML, Button, VBox, HBox, Label, Layout, Select
 from ipywidgets import SelectMultiple, Tab, Text, Textarea, Checkbox
 from ipywidgets import SelectionRangeSlider, Dropdown
 import pandas as pd
 from sqlalchemy import func
+import xarray as xr
 
 import cosima_cookbook
 import cosima_cookbook as cc
-from . import database, querying
+from . import database, querying 
```

Comment:
Re-introduced trailing whitespace :disappointed:! (And many other places through this file)

Reply:
I ran it through black. I may regret that.

Reply:
I am a fan of black! I'd be on board to add it as a pre-merge check.